public/src/components/manual-grading/Release.tsx

```diff
@@ -330,11 +332,12 @@ export class Release extends React.Component<ReleaseProps, ReleaseState>{
             this.setState({
                 open: this.props.isSelected ? !this.state.open : true,
                 reviews: ready,
-                score: totalScore(ready),
+                score: scoreFromReviews(ready),
                 status: this.props.submission?.status ?? Submission.Status.NONE,
             });
         } else {
             this.setState({open: !this.state.open});
         }
     }
+
```

Comment:
Maybe remove this blank line and add a newline at the end of the file.

Reply:
Done.